src/compression_wrapper.c

```diff
@@ -26,6 +26,7 @@
 #include <stdio.h>
 #include <ctype.h>
 #include <stdbool.h>
+#include <unistd.h>
```

Comment:
STDOUT_FILENO is only used if WITH_ZCHUNK macro is defined. Could you please move the #include to an already existing WITH_ZCHUNK block?

Reply:
done